database/ddl/ddl_test.go

```diff
@@ -18,37 +18,37 @@ func TestDDL_NewMap_Postgres(t *testing.T) {
 	// setup types
 	want := &Map{
 		BuildService: &Service{
-			Create: postgres.CreateBuildTable,
+			Create: []string{postgres.CreateBuildTable, postgres.CreatePayloadColumn},
```

Comment:
This is the first time we're introducing Alter sets.
Have we considered separating Create queries and Alter queries? In that way we dont have to reuse Create for both creation and changeset, which might potentially create confusion?

(this does invoke a conversation on how we should do database changesets, which are not currently implemented anywhere....)

Reply:
Or... renaming Create to.... Setup? Not sure yet. Perhaps meant for a separate refactor...

Reply:
I debated on creating a separate section within the `Service` to handle alter sets but thought the ability to specify order within the single slice over weighed separating into multiple slices.